### PR TITLE
Update changelog, bump telemetry snapshot version.

### PR DESCRIPTION
**Pull request by tarigo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Mapbox Android Telemetry
 
+### v8.1.3
+
+- [libtelemetry] Add LocationEvent constructor that doesn't accept permissionStatus [#569](https://github.com/mapbox/mapbox-events-android/pull/569)
+
 ### v8.1.2
 
 - [libtelemetry] Fix LocationEvent timestamp [#567](https://github.com/mapbox/mapbox-events-android/pull/567)
```
Comment (mr1sunshine) [inline CHANGELOG.md:13]: Can we fix the typo on the next line: `ibtelemetry`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Mapbox Android Telemetry
 
+### v8.1.3
```
Comment (mr1sunshine) [inline CHANGELOG.md:7]: just double-checking that we shouldn't bump the Core module version as well, right?

**Review by mr1sunshine — APPROVED** [review]
LGTM, with a couple of questions

File: CHANGELOG.md (hunk 2 above)
Comment (tarigo) [reply CHANGELOG.md:7]: Correct.

**Pull request merged**